io/net/infiniband/ucmatose.py

```diff
@@ -148,11 +161,11 @@ def tearDown(self):
         """
         unset ip
         """
-        if not configure_network.set_mtu_host(self.iface, '1500'):
+        if self.networkinterface.set_mtu('1500') is not None:
             self.fail("Failed to set mtu in host")
-        if not self.peerinfo.set_mtu_peer(self.peer_interface, '1500'):
+        if self.peer_networkinterface.set_mtu('1500') is not None:
             self.fail("Failed to set mtu in peer")
-        configure_network.unset_ip(self.iface)
+        self.networkinterface.remove_ipaddr(self.ipaddr, self.netmask)
```

Comment:
Do not unset the IP.  Let the ip be pingable for the cfg tests.

Reply:
@manvanthar for all test cases ..?